as-vending/res/configuration.yaml

```
@@ -0,0 +1,43 @@
+# Copyright © 2023 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+Writable:
+  LogLevel: INFO
+
+Service:
+  Host: localhost
+  Port: 48099
+  StartupMsg: This microservice checks if ID numbers from REST requests are authenticated
+
+Clients:
+  core-command:
+    Protocol: "http"
+    Host: "localhost"
+    Port: 59882
+
+MessageBus:
+  Optional:
+    ClientId: "app-mqtt-export"
```

Sorry, I copied my suggestion from ASC and didn't change the service name :-(
```suggestion
    ClientId: "as-vending"
```